Makefile.am

```diff
@@ -26,10 +26,14 @@ SUBDIRS = \
 
```

Comment:
commit summary should be more Add support to use GBM & MiniGBM as needed ? sounds like we are removing minigbm support otherwise